lib/global_config.js

```diff
@@ -278,4 +279,8 @@ exports.getKeepAlive = function () {
   return keepAlive;
 };
 
+exports.CredentialManager = CredentialManager;
 
```

Comment:
I think the approach we are looking for is to simplify this a little. Our global config logic could look more like this: 
```
let credentialManager = createJsonFileCredentialManager();

exports.setCustomCredentialManager = function (customCredentialManager) {
  // here we should also add checks if the object actually has these read(), write(), remove() methods. If not then we should throw an error.
  credentialManager = customCredentialManager;
};

exports.getCredentialManager = function () {
  return credentialManager;
};
```
Also note that this method `createJsonFileCredentialManager()` creates a default manager (it's logic currently is in credentialManager.js.

Reply:
So in effect user would set the manager something like this (forgive me my pseudocode skills):
```
setCustomCredentialManager(function () {
  this.read = function (key) {
    return keytar.read(key);
  };
  this.write = function (key, credential) { ... }
  this.remove = function (key) { ... }
});
```
That being said, obviously our default manager set via `createJsonFileCredentialManager()` should have the same interface so they can be used interchangeably.

Reply:
Replaced. I have a question about this. For the local JSON credential, I still left two different files: 'localStorage' and 'credentialManager' just in case when we add different credentialManager methods in the future. However, for now, do you think it would be better to merge those two files?

Reply:
yup, we can merge it.

Reply:
I merged it.